schema/snapcraft.yaml

```diff
@@ -106,6 +106,9 @@ properties:
           stop-command:
             type: string
             description: command executed to stop a service
+          post-stop-command:
+            type: string
+            description: command executed after stopping a service
```

Comment:
oh, wait, I think you are missing the dependencies. Take a look a few lines up:

```
dependencies:
  stop-command: ["daemon"]
```

Just as stop-command, post-stop-command doesn't make sense if this is not a daemon. So there's a missing test here too. Can you please write a test that checks that you can't write a snapcraft.yaml with `post-stop-command` but without `daemon`?